sklearn/metrics/tests/test_common.py

```diff
@@ -303,16 +303,6 @@ def precision_recall_curve_padded_thresholds(*args, **kwargs):
     "micro_average_precision_score",
     "macro_average_precision_score",
     "samples_average_precision_score",
-
```

Comment:
above we still have ``micro_average`` etc for precision?

Reply:
Yes, this list (METRICS_WITH_POS_LABEL) only records metrics with a "pos_label" argument.
Agree that it's trivial, sorry for tagging 0.20.

Reply:
I just didn't understand why this reasoning doesn't apply to ``micro_average_precision`` below, which is not binary, and so doesn't have a pos_label.

Reply:
@amueller For ``average_precision_score``, parameter "average" is ignored when ``y_true`` is binary, so we can also remove micro/weighted/samples_average_score, but that won't make any difference (i.e., we actually run the same test, see ``test_thresholded_invariance_string_vs_numbers_labels``).
The aim of the PR is to get rid of the ``to be removed in 0.18``. This is why I tag it as 0,20.